### ISSUE TITLE
Keyboard navigation doesn't work on Safari

### ISSUE THREAD

**Issue by gnandretta** [auth0/lock-passwordless#46]
The current implementation uses the `tabindex` attribute to allow the user to navigate between the different controls with the _tab_ key. Unfortunately Safari ignores this by default.

![safari-tab-option](https://cloud.githubusercontent.com/assets/120195/10228997/25e34200-684c-11e5-9d1f-3d4db2485eb7.png)

The only solution I could come up with is to listen for the tab keydown event on each input but the trade-offs are significant.
